src/util/config.rs

```diff
@@ -88,6 +102,13 @@ pub struct ChannelHandshakeLimits {
 	/// Defaults to true to make the default that no announced channels are possible (which is
 	/// appropriate for any nodes which are not online very reliably).
 	pub force_announced_channel_preference: bool,
+	/// Set to the amount of time we're willing to wait to claim money back to us.
+	///
+	/// Not checking this value would be also a security issue, as our peer would be able to set it to
+	/// max relative lock-time (a year) and us losing money by useless locked funds.
+	/// Default is MAX_LOCAL_BREAKDOWN_TIMEOUT, we enforce it as a maximum at channel opening offer
```

Comment:
s/, we enforce it as a maximum at channel opening offer/which we also enforce as a maximum value/